packages/wrangler/src/versions/upload.ts

```diff
@@ -538,6 +543,14 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 
 	logger.log("Worker Version ID:", versionId);
 
+	if (versionId && config.workers_dev && hasPreview) {
+		const userSubdomain = await getWorkersDevSubdomain(accountId);
```

Comment:
When thinking more about this - I think I should also be checking the subdomain status of the worker. Version upload doesn't apply the workers_dev setting from the wrangler.toml, so using that as a source of truth could be misleading.

Reply:
Fixed!

Reply:
We may end up wanting to add another warning if workers_dev is out of sync the current output note about "triggers deploy" accurate covers this scenario as well.

Reply:
"Changes to triggers (routes, custom domains, cron schedules, etc) must be applied with the command wrangler triggers deploy"

Though maybe we should specifically call out workers_dev rather than letting it fall under "etc"?